src/main/java/uk/gov/hmcts/reform/roleassignment/domain/service/common/PersistenceService.java

```diff
@@ -245,23 +249,96 @@ public DatabaseChangelogLockEntity releaseDatabaseLock(int id) {
             )
         );
 
+        return prepareQueryRequestResponse(existingFlag, startTime);
```

Comment:
we should avoid passing the start time param and rather remove it from this method or ( get the response from this method and then print the time consumed. This will reduce the reusability of this method.

Reply:
Not getting you.